com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/StepsUtil.java

```diff
@@ -1081,4 +1081,10 @@ public static void setServicesGuids(DelegateExecution context, Map<String, Strin
         }.getType());
     }
 
+    public static boolean getDeleteServicesFlag(ExecutionWrapper execution) {

```

Comment:
By convention names of boolean methods should be questions, so rename this method to "shouldDeleteServices".